bin/summarizePerformanceRst.py

```diff
@@ -66,6 +68,8 @@ def plotMetricScatter(df, name1, name2, band,
         Name of y-axis metric.
     band : `str`
         Filter band to plot.
+    descriptions : `dict` of `str`
+        Descriptions of the names, to inform the plot axes.
```

Comment:
FWIW, even with this added I still wouldn't be able to guess how to populate this dict.

Reply:
Well, I think you wrote the original dict as a global, so I made a guess at what it was.